sql-statements/sql-statement-create-table.md

```diff
@@ -169,7 +169,11 @@ mysql> DESC t1;
 
 ## MySQL compatibility
 
+<<<<<<< HEAD
 * TiDB does not support temporary tables, but it ignores the `CREATE TEMPORARY TABLE` syntax.
+=======
+* TiDB does not support temporary tables. The `CREATE TEMPORARY TABLE` syntax will raise an error if [`tidb_enable_noop_functions = 0`](/system-variables.md#tidb_enable_noop_functions-new-in-v40); TiDB simply ignores the `TEMPORARY` keyword if `tidb_enable_noop_functions = 1`.
+>>>>>>> dccf1a1f... Update compatibility for temporary tables (#5071)
```

Comment:
```suggestion
* TiDB does not support temporary tables. Since v5.0 GA, the `CREATE TEMPORARY TABLE` syntax will raise an error if [`tidb_enable_noop_functions = 0`](/system-variables.md#tidb_enable_noop_functions-new-in-v40); TiDB simply ignores the `TEMPORARY` keyword if `tidb_enable_noop_functions = 1`.
```